packages/api/endpoints/pdrs.js

```diff
@@ -96,4 +114,7 @@ router.get('/:pdrName', get);
 router.get('/', list);
 router.delete('/:pdrName', del);
 
-module.exports = router;
+module.exports = {
+  del,
+  router,
+};
```

Comment:
Not sure why we're exporting `del` separately here. Will likely become clear as I go through the rest 😄

Reply:
Exporting the endpoint handler directly allows us to do some mock injection in unit testing using `req.testContext` and more efficiently test all the different delete cases